.github/workflows/pre-main.yaml

```diff
@@ -226,47 +218,35 @@ jobs:
 
       # Perform smoke tests.
       - name: 'Test: Run test suites'
-        run: TNF_LOG_LEVEL=${TNF_SMOKE_TESTS_LOG_LEVEL} ./run-cnf-suites.sh -l "${SMOKE_TESTS_LABELS_FILTER}"
+        run: ./certsuite run --label-filter="${SMOKE_TESTS_LABELS_FILTER}" --output-dir="certsuite-out" --log-level=${SMOKE_TESTS_LOG_LEVEL}
```

Comment:
No need quotes for `certsuite-out` and need for `${SMOKE_TESTS_LOG_LEVEL}`.

Reply:
Done.